android/src/main/java/co/apptailor/googlesignin/RNGoogleSigninModule.java

```diff
@@ -135,12 +135,12 @@ public void configure(
     }
 
     @ReactMethod
-    public void signInSilently(Promise promise) {
```

Comment:
Please keep the previous name - signInSilently! Thanks

Reply:
resolved in 6b1632ef